packages/tpetra/core/test/CrsGraph/CrsGraph_UnitTests_Swap.cpp

```diff
@@ -125,55 +125,58 @@ TEUCHOS_STATIC_SETUP()
 // gbl_num_columns: Max # of columns in the matrix-representation of the graph.
 //                  This should be >= the highest value of v from all edges (u,v) in edges.
 //                  Note: u and v are 0-indexed, so if the highest v is 11, then this should be 12.
-template<class LO, class GO, class Node, class comm_t>
+template<class LO, class GO, class Node, class comm_type>
```

Comment:
1. You don't need to template on `comm_type` here.  It's always `const Teuchos::Comm<int>`.
2. Typedefs are snake case, template parameters are Capitalized or CamelCase.  See `std::map` for examples: https://en.cppreference.com/w/cpp/container/map

Reply:
template on `comm_type` removed.